src/mixins/filing-template-mixin.ts

```diff
@@ -4,9 +4,11 @@ import { Getter, Action } from 'vuex-class'
 import { DateMixin } from '@/mixins'
 
 // Interfaces
-import { ActionBindingIF, BusinessContactIF, CertifyIF, DateTimeIF, DefineCompanyIF,
-  IncorporationAgreementIF, IncorporationFilingIF, NameTranslationIF, PeopleAndRoleIF,
-  ShareStructureIF } from '@/interfaces'
+import {
+  ActionBindingIF, BusinessContactIF, CertifyIF, CreateRulesIF, DateTimeIF, DefineCompanyIF,
+  IncorporationAgreementIF, IncorporationFilingIF, NameTranslationIF, PeopleAndRoleIF, RulesDocIF,
+  ShareStructureIF
+} from '@/interfaces'
```

Comment:
Use type "CreateRulesIF" for rules on line 38.

Reply:
updated to use `CreateRulesIF`